NYPLAudiobookToolkit/Utils/JSONUtils.swift

```diff
@@ -0,0 +1,73 @@
+class JSONUtils {
+    enum JSONCanonicalizatoinError: Error {
+        case canonicalizationError(String)
+    }
+    static let hexArray = [
+        "00", "01", "02", "03", "04", "05", "06", "07", "08", "09", "0A", "0B", "0C", "0D", "0E", "0F",
+        "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "1A", "1B", "1C", "1D", "1E", "1F"
+    ]
+    class func canonicalize(jsonObj: Any?) throws -> String {
+        return try canonicalizeInternal(jsonObj: jsonObj)
+    }
+    
+    private class func canonicalizeInternal(jsonObj: Any?) throws -> String {
```

Comment:
can you add some comments explaining what canonicalize actually means here?

Reply:
Description added to the calling function